src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -0,0 +1,433 @@
+---
+title: Managing Several Displays with the Multi-Screen Window Placement API
+subhead: Get information about connected displays and position windows relative to those displays.
+authors:
+  - thomassteiner
+description:
+  The Multi-Screen Window Placement API allows you to enumerate the displays connected to your
+  machine and to place windows on specific screens.
+date: 2020-09-03
+# updated: 2020-09-03
+tags:
+  - blog
+  - capabilities
+hero: hero.jpg
+alt: Simulated trading desk showing multiple fake cryptocurrencies and their price charts.
+origin_trial:
+  url: https://developers.chrome.com/origintrials/#/view_trial/1411878483180650497
+---
+
+{% Aside %} The Multi-Screen Window Placement API is part of the
+[capabilities project](https://developers.google.com/web/updates/capabilities) and is currently in
```

Comment:
I thought we use https://web.dev/fugu-status/ for the "capabilities project" link. See https://web.dev/serial/ for instance

Reply:
This came straight from our template. I have updated the template accordingly now.